core/src/main/java/org/apache/carbondata/core/metadata/schema/table/TableSchemaBuilder.java

```diff
@@ -75,6 +78,11 @@ public TableSchemaBuilder tableName(String tableName) {
     return this;
   }
 
+  public TableSchemaBuilder resetTransactionalTable(boolean isTransactionalTable) {
```

Comment:
change function name to setTransactionTable , there nothing resetting happening in this function